src/plugins/workspace/public/components/home_get_start_card/setup_get_start_card.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React from 'react';
+import { CoreStart } from 'opensearch-dashboards/public';
+import { EuiIcon } from '@elastic/eui';
+import { BehaviorSubject } from 'rxjs';
+import { i18n } from '@osd/i18n';
+import {
+  ContentManagementPluginStart,
+  HOME_CONTENT_AREAS,
+} from '../../../../content_management/public';
+import { WorkspaceUseCase } from '../../types';
+import { getUseCaseFromFeatureConfig, getUseCaseUrl } from '../../utils';
+import { UseCaseCardTitle } from './use_case_card_title';
+
+const createContentCard = (useCase: WorkspaceUseCase, core: CoreStart) => {
+  const { workspaces, application, http } = core;
+  const workspaceList = workspaces.workspaceList$.getValue();
```

Comment:
I suggest we moving the logic to the component and listen to worksapceList$ change.

Reply:
I tried to implement this change, but I couldn't. The return in render is `react.jsx.element`, and in this logic, object is also returned, so the logic cannot be coupled together.